engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -447,6 +447,30 @@ FLUTTER_DARWIN_EXPORT
 
 @end
 
+/**
+ * Exposes parts of a `FlutterEngine` for registration purposes.
+ *
+ * This is used when the engine is created implicitly to allow registering
+ * plugins, application-level method channels, platform views, etc.
+ */
+@interface FlutterImplicitEngineBridge : NSObject
+
+@property(nonatomic, strong) NSObject<FlutterPluginRegistry>* pluginRegistry;
+@property(nonatomic, strong) NSObject<FlutterApplicationRegistrar>* applicationRegistrar;
+
+@end
+
+/**
+ * Protocol for receiving a callback when an implicit engine is initialized.
```

Comment:
Maybe expand more on what an implicit engine is here? I'm not sure the average Flutter developer (even restricting to the subset who are interacting with the native app code) will understand what an implicit engine is.